pages/dkp/kommander/1.3/clusters/index.md

```diff
@@ -153,13 +153,58 @@ Figure 1. A cluster card with resources
 
 For more detailed information, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) about resources.
 
-## Platform Services
+## Platform services
 
 Services that have been installed on your management cluster. You can visit a cluster's detail page to see which platform services have been enabled under the "Addons" section.
 
 ![Cluster Detail Page](/dkp/kommander/1.3/img/cluster-detail-page.png)
 
 Figure 2. Cluster detail page
 
+### Custom service cards
+
+Custom service cards can be added to the cluster detail page's Addons section by creating a `ConfigMap` on the cluster. The `ConfigMap` must have a `d2iq.io/addon` label and must contain both `name` and `dashboardLink` data keys to be displayed. Upon creation of the `ConfigMap` the GUI will show a card corresponding to the data provided in the `ConfigMap`. Custom cards have a Kubernetes icon and can link to a service running in the cluster or use an absolute URL to link to any accessible URL.
```

Comment:
Do we want to say "GUI" here or is "UI" fine? @TattdCodeMonkey 

FYI - suggestion unrelated to the above comment

```suggestion
Custom service cards can be added to the cluster detail page's Addons section by creating a `ConfigMap` on the cluster. The `ConfigMap` must have a `d2iq.io/addon` label and must contain both `name` and `dashboardLink` data keys to be displayed. Upon creation of the `ConfigMap`, the GUI will show a card corresponding to the data provided in the `ConfigMap`. Custom cards have a Kubernetes icon and can link to a service running in the cluster or use an absolute URL to link to any accessible URL.
```